examples/with-firebase/firebase/nodeApp.js

```diff
@@ -0,0 +1,17 @@
+import * as admin from 'firebase-admin'
+
+if (!admin.apps.length) {
+  admin.initializeApp({
+    credential: admin.credential.cert({
+      projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
+      clientEmail: process.env.NEXT_PUBLIC_FIREBASE_CLIENT_EMAIL,
+      privateKey: process.env.NEXT_PUBLIC_FIREBASE_PRIVATE_KEY.replace(
+        /\\n/g,
+        '\n'
+      ),
+    }),
+    databaseURL: process.env.NEXT_PUBLIC_FIREBASE_DATABASE_URL,
```

Comment:
These keys should not start with `NEXT_PUBLIC_` as they're not be public, they are also used in Node.js for SSR so accessing them in the browser won't be required.

Reply:
Changed to `FIREBASE_CLIENT_EMAIL` `FIREBASE_PRIVATE_KEY`.
`NEXT_PUBLIC` prefix was needed for other environment variables for use with `clientApp`.